src/Components/Scenes/Hydration/NewJob/NewJob.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import NewJobHeader from './NewJobHeader';
+
+const chromeStyles = makeStyles(theme => ({
+  container: {
+    position: 'fixed',
+    zIndex: 1101, // Positions container on top of original header bar
```

Comment:
Do you think it will be better to conditional render the appbar based on the pathname that way we don't have to use zIndex. Or we could have a hydration header component. What are your thoughts?

Reply:
Yeah, I was super conflicted about this as well! I was trying to avoid having any messy conditional render logic within the higher-level PageWrapper component that was dependent on routing, but I definitely agree zIndex is gross haha. My rationale was that there's also going to be a good amount of interactivity with the header, see: [here](https://www.figma.com/file/jTZDpSj1DmDmwZ9o4IWGiI/Upload-Scripts?node-id=1%3A1198). Additionally, we need the canvas to fill the entirety of the page.

I'm thinking something in PageWrapper like: `{currentPath === '/hydration/new-job' ? null : <AppBarContainer />}` -- that way we don't have to rely on zIndex, but we're still able to have a custom header for the NewJob view. Lmk what you think!

Reply:
That's exactly what I was thinking. 🔥